### PR TITLE
Create application modal improvement

### PR DESCRIPTION
**Pull request by y-kkamil**
**Description**

Changes proposed in this pull request:

- Scenario chooser fix
- Adjust modal's width

**Related issue(s)**
See #1401

### CODE REVIEW

File: compass/src/components/Applications/CreateApplicationModal/styles.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .margin-top {
   margin-top: 15px;
 }
+
+.fd-modal {
```
Comment (Wawrzyn321) [inline compass/src/components/Applications/CreateApplicationModal/styles.scss]: Please use more specific selector, as this one affects other modals, for example the one at `ApplicationDetails::Connect Application`.

File: compass/src/components/Applications/CreateApplicationModal/CreateApplicationModal.component.js [hunk 2]
```diff
@@ -35,15 +37,22 @@ class CreateApplicationModal extends React.Component {
...
       formData: {
         ...this.state.formData,
-        labels: scenarios && scenarios.length ? { scenarios } : {},
+        labels:
+          selectedScenarios && selectedScenarios.length
+            ? { scenarios: selectedScenarios }
+            : {},
       },
+      scenariosToSelect: scenariosToSelect,
+      selectedScenarios: selectedScenarios,
```
Comment (Wawrzyn321) [inline compass/src/components/Applications/CreateApplicationModal/CreateApplicationModal.component.js]: Can we use shorthand there?

Comment (y-kkamil) [reply compass/src/components/Applications/CreateApplicationModal/CreateApplicationModal.component.js]: of course we can, corrected.

File: compass/src/components/Applications/CreateApplicationModal/styles.scss (hunk 1 above)
Comment (y-kkamil) [reply compass/src/components/Applications/CreateApplicationModal/styles.scss]: :+1:

**Pull request merged**